docs/examples/gallery/chisquared.md

```diff
@@ -0,0 +1,93 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+# Chi-Squared Distribution
+
+The chi-squared (also chi-square or $\chi^2$) is a continuous probability distribution characterized by a single parameter, $\nu$, usually called degrees of freedom. This distribution emerges from the sum of the squares of $\nu$ independent standard normal random variables.
+
+The chi-squared distribution is widely used in many statistical tests, for hypothesis testing and constructing confidence intervals. 
+
+## Probability Density Function (PDF):
+
+```{code-cell}
+---
+tags: [remove-input]
+mystnb:
+  image:
+    alt: Chi-Squared Distribution PDF
+---
+
+import arviz as az
+from preliz import ChiSquared
+az.style.use('arviz-doc')
+nus = [1, 3, 9]
+
+for nu in nus:
+    ax = ChiSquared(nu).plot_pdf(support=(0, 20))
```

Comment:
Maybe this will do

```suggestion
import numpy as np
import arviz as az
from preliz import ChiSquared
az.style.use('arviz-doc')
nus = [1, 3, 9]

for nu in nus:
    ax = ChiSquared(nu).plot_pdf(support=(np.finfo(float).eps, 20))
```